activities/models/post.py

```diff
@@ -336,6 +360,20 @@ def to_create_ap(self):
             "object": object,
         }
 
+    def to_update_ap(self):
+        """
+        Returns the AP JSON to update this object
+        """
+        object = self.to_ap()
+        return {
+            "to": object["to"],
+            "cc": object.get("cc", []),
+            "type": "Update",
+            "id": self.object_uri + "#update",
+            "actor": self.author.actor_uri,
+            "object": object,
```

Comment:
We also need to update `to_ap` to transform the local `edited` field into the AP `updated` field if it's present.